### PR TITLE
[plugin.video.ytchannels] 0.2.1+matrix.1

### PR DESCRIPTION
**Pull request by mintsoft**
### Description

Support for Matrix

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Review by basrieter — DISMISSED** [review]

File: plugin.video.ytchannels/resources/lib/functions.py [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+		if decoded_data['items'][i]['kind']=='youtube#playlist':
+			playlist_id=decoded_data['items'][i]['id']
+			playlist_name=decoded_data['items'][i]['snippet']['title']
+			thumb=decoded_data['items'][i]['snippet']['thumbnails']['high']['url']
+
+			playlists.append([playlist_id,playlist_name,thumb])
+	return playlists
+
+def local_string(string_id):
+	return my_addon.getLocalizedString(string_id)
```
Comment (enen92) [inline plugin.video.ytchannels/resources/lib/functions.py:259]: missing line at the end of the file

File: plugin.video.ytchannels/default.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+from resources.lib.ytchannels import ytchannels_main
+
+ytchannels_main()
```
Comment (enen92) [inline plugin.video.ytchannels/default.py:4]: missing new line at the end of the file

**Review by enen92 — APPROVED** [review]

**Comment by enen92** [comment]
Thanks for moving forward to matrix :)

**Comment by mintsoft** [comment]
Thanks for the merge! @enen92

**Pull request merged**